common/brave_content_client.cc

```diff
@@ -12,9 +12,11 @@ BraveContentClient::BraveContentClient() {}
 BraveContentClient::~BraveContentClient() {}
 
 void BraveContentClient::AddAdditionalSchemes(Schemes* schemes) {
+#if !defined(OS_ANDROID)
```

Comment:
not sure how this is related to rewards

Reply:
this will also cause problems with brave-core android

Reply:
Removed